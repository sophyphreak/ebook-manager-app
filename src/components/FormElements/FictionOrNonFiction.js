import React, { Component } from 'react';
import RadioHorizontal from './MainElements/RadioHorizontal';

const FictionOrNonFiction = (props) => (
  <RadioHorizontal
    label=""
    options={['Non-fiction', 'Fiction']}
    {...props}
  />
);

export default FictionOrNonFiction;