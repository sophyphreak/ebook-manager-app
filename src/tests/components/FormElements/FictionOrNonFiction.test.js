import React from 'react';
import { shallow } from 'enzyme';
import FictionOrNonFiction from '../../../components/FormElements/FictionOrNonFiction';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<FictionOrNonFiction />);
});

test('should render FictionOrNonFiction correctly', () => {
  expect(wrapper).toMatchSnapshot();
});