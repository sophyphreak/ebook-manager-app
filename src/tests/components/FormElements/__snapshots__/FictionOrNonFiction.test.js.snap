// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FictionOrNonFiction correctly 1`] = `
<RadioHorizontal
  label=""
  options={
    Array [
      "Fiction",
      "Non-fiction *",
    ]
  }
/>
`;
